revivelib/core.go

```diff
@@ -163,7 +163,7 @@ func (r *Revive) Format(
 	<-exitChan
 
 	if formatErr != nil {
-		return "", exitCode, fmt.Errorf("formatting: %w", err)
+		return "", exitCode, fmt.Errorf("formatting: %w", formatErr)
```

Comment:
Could you also update the tests in `core_test.go`?

Reply:
ok, I will try add one test for it.

Reply:
I research the revive, and can't add a test. all formatter is always return nil